pages/components/Navigation/navigation.css

```diff
@@ -0,0 +1,48 @@
+.nav {
```

Comment:
For mobile devices, you can put the logo on top-center and links beneath it 
![image](https://user-images.githubusercontent.com/45557594/112745673-d7fedc80-8fc7-11eb-94ef-973fcb0901b3.png)

![image](https://user-images.githubusercontent.com/45557594/112745697-0381c700-8fc8-11eb-8b78-bced99dacfc7.png)